### ISSUE TITLE
Recaf context menu shows gigantic black box on extremely long identifier

### ISSUE THREAD

**Issue by cyoung06** [Col-E/recaf-3x-issues#105]
```
java.lang.RuntimeException: Requested texture dimensions (110880x330) require dimensions (0x330) that exceed maximum texture size (16384)
        at javafx.graphics/com.sun.prism.es2.ES2RTTexture.create(ES2RTTexture.java:220)
        at javafx.graphics/com.sun.prism.es2.ES2ResourceFactory.createRTTexture(ES2ResourceFactory.java:165)
        at javafx.graphics/com.sun.javafx.tk.quantum.UploadingPainter.run(UploadingPainter.java:125)
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
        at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
        at javafx.graphics/com.sun.javafx.tk.RenderJob.run(RenderJob.java:58)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at javafx.graphics/com.sun.javafx.tk.quantum.QuantumRenderer$PipelineRunnable.run(QuantumRenderer.java:126)
        at java.base/java.lang.Thread.run(Thread.java:833)
```

Above is the related exception.

I have a jar obfuscated with a commercial tool, and it contains extremely long identifiers (like 19000 space characters...)

Right-clicking to view the context menu reveals a gigantic black box
![image](https://user-images.githubusercontent.com/42869671/183286125-ef2e58e3-5e16-4edd-bfd7-098de7304fe6.png)

I can still access the menu by hovering over a random position, but it would be great if recap did some kind of thing to prevent the gigantic black box.

By the way, this behavior is present in recaf 2.x versions too

**Comment by Col-E** [comment]
Looks like name length filtering should be done here. Ez fix.

**Comment by Col-E** [comment]
Fixed in 7f47efa4eee9dd34acebe66ff315ce2304c0d51f
